readthedocs/templates/errors/404/no_subproject.html

```diff
@@ -0,0 +1,22 @@
+{% extends "errors/404/base.html" %}
```

Comment:
Isn't this template the same as `no_project_page.html`? If so, we should extend that one. Otherwise, change the content to reflect that this is a subproject; in case it gives the user more context, but I don't think so...